### PR TITLE
Remove model definition file when deleting by id

### PR DESCRIPTION
**Pull request by superkhau**

### CODE REVIEW

**Comment by superkhau** [comment]
@raymondfeng Review pls.

**Comment by superkhau** [comment]
Also, I'm not sure if fire and forget is a good solution, but its the quickest to impl ATM. I figure you might want me to do something like async.parallel the 2 deletes and send a fail callback if anyone fails. Let me know the preferred solution if this MVP one isn't good enough.

**Comment by superkhau** [comment]
@slnode test please

File: common/models/model-definition.js [hunk 1]
```diff
@@ -125,6 +125,10 @@ function ready(ModelDefinition) {
       removeById(id, function(err) {
         if (err) return cb(err);
 
+        // fire and forget since we want to remove the other files even if this
+        // one doesn't exist
+        fs.unlink(modelDef.getScriptPath());
```
Comment (raymondfeng) [inline common/models/model-definition.js]: I think `fs.unlink()` takes a callback as the 2nd argument.

**Comment by raymondfeng** [comment]
@superkhau You can delete things in parallel or series.

**Comment by superkhau** [comment]
@raymondfeng Review again pls.

**Comment by raymondfeng** [comment]
LGTM

**Pull request merged**